config/custom-environment-variables.json

```diff
@@ -38,7 +38,7 @@
       "caseAssignmentApi": "SERVICES_CCD_CASE_ASSIGNMENT_API"
     },
     "hearings": {
-      "hearingsJurisdictions": "SSCS,PRIVATELAW,CIVIL,IA",
+      "hearingsJurisdictions": "JURISDICTIONS",
```

Comment:
Wrong env variable name, need to create a new one called "HEARINGS_JURISDICTIONS"